tests/lib/rules/comma-dangle.js

```diff
@@ -392,8 +392,8 @@ ruleTester.run("comma-dangle", rule, {
                 {
                     message: "Missing trailing comma.",
                     type: "Property",
-                    line: 2,
-                    column: 11
+                    line: 3,
+                    column: 1
```

Comment:
> I think the original reported location in this case makes more logical sense for the user, since that's where he will have to insert the trailing comma.

As a nitpick, this is only true if the user is using `comma-style: last`. If the user formatted their commas like this, then reporting the closing bracket would make more sense:

```js
var foo = {
  bar: 'baz'
  , qux: 'foobar'
};
^
```

However, in general I would say you're probably right, since I think `comma-style: last` is much more common.